packages/data-point-express/lib/route-map.test.js

```diff
@@ -71,7 +71,7 @@ describe('toCollection', () => {
 })
 
 describe('sortByPriority', () => {
-  test('sorts routes in prioroty order', () => {
+  test('sorts routes in prioroty order according to priority key', () => {
```

Comment:
Maybe change to `sorts routes in prioroty order according to priority key`?

Either way, the first `priority` is misspelled